encrypted-signing-method/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import { Transaction }        from './transaction';
+import { SigningMethodInput } from './models';
+
+export function main() {
+    testEncryptedSigningMethod()
+        .then(() => console.log('done'));
+}
+
+async function testEncryptedSigningMethod() {
```

Comment:
You only need the `async` keyword when you're using `await` in the function. So you can either use `async` + `await` or just use `.then()` on its own.

Technically both do the same thing, but async + await is more commonly used nowadays.